README.md

```diff
@@ -107,6 +138,60 @@ Plugin is closely integrated with quickfix list and allows to use quickfix featu
 Bug reports, feedback, feature and other pull requests are appreciated. Check the [Contributing Guidelines](CONTRIBUTING.md) for how to
 create a feature request, submit a pull request or post an issue.
 
+## **Tips and tricks**
+
+### ccmake
+
+If you want to run `ccmake` in the build directory, you can do something like this:
+```
+nnoremap <expr> <leader>cc ":tab terminal ++close ccmake " . utils#cmake#getBuildDir() . "\<CR>"
```

Comment:
It looks good.
Maybe we can add CCMake command to the plugin.
I think it can be added in the separate PR.

What do you think?

Reply:
We could add this as a command, yes.

The reason I created `Tips and tricks` section is because a lot of these things are quite simple to write, and maybe people would like to modify them so I just gave an example instead of creating a command. Some would like it to appear in a new `tab`, some would like a `vertical split`, etc.

Reply:
Yes I agree, and it is a good idea to have such section in the readme.
But also CCMake support in this case looks good to me! I have tried this command, looks good. If we introduce the CCMake command, I think the argument for this command can allow to configure how to open it.

Reply:
Sure, in the next PR then!

I have another one ready for `g:cmake_run_target_args` and more `vimspector` integration so I can put that there :)